content/millennium/r4/clinical/care-provision/care-plan.md

```diff
@@ -60,19 +60,19 @@ _Implementation Notes_
  `patient`    | See notes | [`reference`] | Who care plan is for. Example: `patient=12345`
  `category`   | See notes | [`token`]     | The scope of care plan being searched for. Examples: `category=assess-plan`
  [`_count`]   | N         | [`number`]    | Number of results per page.
- `_revinclude`| No        | [`token`]     | Provenance resource entries to be returned as part of the bundle. Example:_revinclude=Provenance:target
+ `_revinclude`| N         | [`token`]     | Provenance resource entries to be returned as part of the bundle. Example:_revinclude=Provenance:target
 
 
 Notes:
 
 - The `_id` parameter
-  - May not be provided with any other parameters.
+  - May not be provided with any other parameters, except for `_revinclude` as indicated below.
   - When provided, `_count` is ignored.
 
 - The `date` parameter
-  - When provided, must use both `ge` and `le` prefixes in the same search
-    - The lower value must have the `ge` prefix and the higher value must have the `le` prefix
-    - If date precision must be consistent
+  - When provided, must use both `ge` and `le` prefixes in the same search.
+    - The lower value must have the `ge` prefix and the higher value must have the `le` prefix.
+    - If date precision must be consistent.
```

Comment:
is "If" intentional?

Reply:
No.
Adjusted in latest commit to the branch
![image](https://github.com/cerner/fhir.cerner.com/assets/144141792/65e3b513-2155-4509-bb42-8420da00516c)